### PR TITLE
Fix variables when used in an embedded window like on backstage

### PR DESCRIPTION
**Pull request by ElementalWarrior**
These variables are not properly defined when using mkdocstrings on a backstage website.

### CODE REVIEW

**Comment by ElementalWarrior** [comment]
@pawamoy to hazard nagging you more. Could you review this as well.

I don't think the test failure in path based tests could be related to this simple css selector change.

**Comment by pawamoy** [comment]
Hey @ElementalWarrior, thanks for the PR! Can you explain to me what is this `:host` variable and why it's required for Backstage? Is it a standard thing, or specific to Backstage? Seems like a harmless change anyway, just want to understand.

**Comment by pawamoy** [comment]
OK thanks. I don't know enough of HTML/CSS so I'll trust you on this. We can always revert if this causes any issue :slightly_smiling_face:

**Review by pawamoy — APPROVED** [review]

File: docs/css/material.css [hunk 1]
```diff
@@ -4,7 +4,7 @@
 }
 
 /* Custom admonition: preview */
-:root {
+:root, :host {
```
Comment (pawamoy) [inline docs/css/material.css:7]:
```suggestion
:root {
```

File: docs/usage/customization.md [hunk 2]
```diff
@@ -127,7 +127,7 @@ if it supports light and dark themes,
 otherwise just override the variables at root level:
 
 ```css title="docs/css/mkdocstrings.css"
-:root {
+:root, :host {
```
Comment (pawamoy) [inline docs/usage/customization.md:130]:
```suggestion
:root {
```

**Review by pawamoy — COMMENTED** [review]
I'll just remove the `:host` occurrences from our own docs.

**Pull request merged**